pkg/controller/core/workload_controller.go

```diff
@@ -164,7 +164,7 @@ func (r *WorkloadReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 			case kueue.WorkloadEvictedByDeactivation:
 				workload.SetRequeuedCondition(&wl, kueue.WorkloadReactivated, "The workload was reactivated", true)
 				updated = true
-			case kueue.WorkloadEvictedByPodsReadyTimeout:
+			case kueue.WorkloadEvictedByPodsReadyTimeout, kueue.WorkloadEvictedByAdmissionCheck:
```

Comment:
I'm wondering if this change belong conceptually to the same fix.
IIUC the intention of this change (and the one below 
```
// The requeued condition status set to true only on EvictedByPreemption
setRequeued := evCond.Reason == kueue.WorkloadEvictedByPreemption
```
IIUC they are aiming to "respect backoff delay when requeuing due to failed admission checks", and could be a separate bugfix. Let me know if I'm missing something.

Reply:
That's right, fixed